index.js

```diff
@@ -1,74 +1,228 @@
-const WEIS_IN_ETH = 1e18;
-const WEIS_IN_GWEI = 1e9;
-const GWEIS_IN_ETH = 1e9;
-
-const SATOSHIS_IN_BTC = 1e8;
-
-const WEIS_IN_SATOSHI = WEIS_IN_ETH / SATOSHIS_IN_BTC;
+const bigDecimal = require('js-big-decimal');
+
+const WEIS_TO_GWEI_PRECISION = 9;
+const WEIS_TO_ETH_PRECISION = 18;
+const GWEIS_TO_ETH_PRECISION = 9;
+const SATOSHIS_TO_BTC_PRECISION = 8;
+
+const WEIS_IN_ETH = 10 ** WEIS_TO_ETH_PRECISION;
+const WEIS_IN_GWEI = 10 ** WEIS_TO_GWEI_PRECISION;
+const GWEIS_IN_ETH = 10 ** GWEIS_TO_ETH_PRECISION;
+const SATOSHIS_IN_BTC = 10 ** SATOSHIS_TO_BTC_PRECISION;
+
+const WEIS_IN_SATOSHI = bigDecimal.divide(WEIS_IN_ETH, SATOSHIS_IN_BTC);
+const MINIMUM_WEIS_TO_SATOSHIS = 10000000000;
```

Comment:
This is basically WEIS_IN_SATOSHI, right? Why not use that value directly? It's understandable that that's the minimum. Just 1 satoshi represented in weis

Reply:
Updated!